FIAT/reference_element.py

```diff
@@ -1467,14 +1438,14 @@ def contains_point(self, point, epsilon=0):
         return self.product.contains_point(point, epsilon=epsilon)
 
     def distance_to_point_l1(self, point, rescale=False):
-        """Get the L1 distance (aka 'manhatten', 'taxicab' or rectilinear
+        """Get the L1 distance (aka 'manhattan', 'taxicab' or rectilinear
         distance) to a point with 0.0 if the point is inside the cell.
 
         For more information see the docstring for the UFCSimplex method."""
         return self.product.distance_to_point_l1(point, rescale=rescale)
 
     def symmetry_group_size(self, dim):
-        return [1, 2, 8][dim]
+        return [1, 2, 8, 48][dim]
```

Comment:
Not entirely sure where these numbers come from, but maybe the general formula is `2**dim * factorial(dim)`

Reply:
I think you added 48 - I'll look into it, would definitely be better to have a general formula.